### ISSUE TITLE
VMware Fusion 5 Support

### ISSUE THREAD

**Issue by jasonberanek** [thbishop/fission#23]
Operating through [veewee](https://github.com/jedi4ever/veewee), VMware Fusion 5.0.2 Professional behaves as expected with fission as written. The one exception is VM deletion which returns the following error because the properties list for Fusion changed with version 5:

```
fission-0.4.0/lib/fission/metadata.rb:88:in `delete_vm_favorite_entry': undefined method `delete_if' for nil:NilClass (NoMethodError)
    from fission-0.4.0/lib/fission/metadata.rb:27:in `delete_vm_info'
    from fission-0.4.0/lib/fission/vm.rb:675:in `delete'
```

The underlying files are removed, but the metadata isn't erased leaving the VM in the Fusion Virtual Machine Library list. Time allowing, I will try to investigate a solution and submit a pull request accordingly.

**Comment by jedi4ever** [comment]
@thbishop could you merge it in?

**Comment by thbishop** [comment]
i've pushed up 0.5.0.beta.1 to ruby gems which includes this. give it a whirl and let me know if you run into any issues.

**Comment by jedi4ever** [comment]
thx @thbishop

**Comment by mrinalwadhwa** [comment]
@jedi4ever do you plan on updating veewee.gemspec to support the above version of fission soon?

**Comment by thbishop** [comment]
i've released 0.5.0
